### DISCUSSION TITLE
file manager

### DISCUSSION THREAD

**Discussion by ghost** [pocketbase/pocketbase#5485] [Polls]
Would you find it valuable to have a file manager feature in PocketBase to view images and directories? 👍

**Comment by ganigeorgiev** [comment]
Polls like the above are pointless if there is nothing concrete to work on or discuss.

In general there are no plans for "file manager" as it is not clear how this would work since each collection can have one or many `file` type fields (and each `file` field could be single or multiple).

You can always emulate "file manager" by creating a "Media" collection with a single `file` field.

Note also that the `editor` field type has already a file picker that works with all collections with at least 1 `file` field.

**Comment by ealbinu** [comment]
I’m currently working with a closed Pocketbase instance where I don’t have access to the files. The main challenge has been working with the hooks (/pb_hooks/<...>.js). It would be ideal to access ‘extension files’ without needing to work on another instance and redeploy, all from within the dashboard.
